content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -624,23 +671,35 @@ You can also use `kubectl rollout status sts/<name>` to view
 the status of a rolling update to a StatefulSet
 {{< /note >}}
 
-#### Staging an Update
+#### Staging an update
 
-You can stage an update to a StatefulSet by using the `partition` parameter of
-the `RollingUpdate` update strategy. A staged update will keep all of the Pods
-in the StatefulSet at the current version while allowing mutations to the
-StatefulSet's `.spec.template`.
+You can split updates to a StatefulSet that uses the `RollingUpdate` strategy
+into _partitions_, by specifying `.spec.updateStrategy.rollingUpdate.partition`.
 
-Patch the `web` StatefulSet to add a partition to the `updateStrategy` field:
+For more context, you can read [Partitioned rolling updates](/docs/concepts/workloads/controllers/statefulset/#partitions)
+in the StatefulSet concept page.
+
+You can stage an update to a StatefulSet by using the `partition` field within
+`.spec.updateStrategy.rollingUpdate`.
+For this update, you will keep the existing Pods in the StatefulSet using
+unchanged whilst you change the pod template for the StatefulSet.
```

Comment:
"using unchanged" ???

Reply:
I'll mark this resolved. Let me know if that doesn't sound right.